### PR TITLE
Change readme file

### PR DESCRIPTION
**Pull request by Murshadhaque**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hmcts-kodatala — APPROVED** [review]
OK

File: config/default.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ services:
     systemUsername: 'dummy_user'
     systemPassword: 'dummy_password'
   fis:
-    url: 'http://fis-cos-api-pr-80.service.core-compute-preview.internal'
+    url: 'https://fis-cos-api-pr-80.service.core-compute-preview.internal'
```
Comment (adusumillipraveen) [inline config/default.yaml:18]: you should make these env vars and handle as env vars in charts and locals. This is not sustainable.

Comment (adusumillipraveen) [inline config/default.yaml:18]:
```suggestion
    url: 'http://fis-cos-api-pr-80-java'
```

File: config/development.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ services:
     microservice: 'ds_ui'
     secret: 'NEED TO INSERT SECRET'
   fis:
-    url: 'http://fis-cos-api-pr-80.service.core-compute-preview.internal'
+    url: 'https://fis-cos-api-pr-80.service.core-compute-preview.internal'
```
Comment (adusumillipraveen) [inline config/development.yaml:11]:
```suggestion
    url: 'http://fis-cos-api-pr-80-java'
```

**Review by yogendraU — APPROVED** [review]

**Review by Shilpa-Krishna — APPROVED** [review]

**Pull request merged**